### ISSUE TITLE
Just doesn't work.

### ISSUE THREAD

**Issue by shepsii** [jay2503/cordova-plugin-ios-in-app-ratings-and-reviews#3]
I just get requestReview is not defined. Why isn't there a usage example?

Anyway, suggest everyone uses this plugin: https://github.com/dpa99c/cordova-launch-review#usage as for one it actually works!

**Comment by rsanchez** [comment]
For anyone else reading this:

```
cordova.plugins.InAppRatingsReview.prototype.requestReview();
```

or

```
var InAppRatingsReview = new cordova.plugins.InAppRatingsReview();

InAppRatingsReview.requestReview();
```

**Comment by jay2503** [comment]
fixed with PR merged #4
